.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - "**"
```

Comment:
This only applies to branches that are forked from `legacy`, which is based on `v1.1.0`.

Reply:
Confusing. Gross. Okay.

Reply:
Removed this because it's unnecessary -- without `branches:` defined it defaults to `- "**"` anyway.